### PR TITLE
[garp_service]: Make module scoped

### PR DESCRIPTION
**Pull request by theasianpianist**
Signed-off-by: Lawrence Lee <lawlee@microsoft.com>

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
Different test modules may configure garp_service differently.

#### How did you do it?

* Make fixture module scoped
* Move dualtor common setup fixture to dualtor/conftest.py and only run garp_service fixture for true dual ToR testbeds.

#### How did you verify/test it?
Run dual ToR test cases and verify they pass

### CODE REVIEW

File: tests/dualtor/test_normal_op.py [hunk 1]
```diff
@@ -12,11 +12,6 @@
 ]
 
 
-@pytest.fixture(scope="session", autouse=True)
-def common_setup_teardown(run_arp_service):
```
Comment (vaibhavhd) [inline tests/dualtor/test_normal_op.py:16]: I think we never had `run_arp_service` fixture. Is that why this is now being removed?

**Review by vaibhavhd — APPROVED** [review]

File: tests/dualtor/test_normal_op.py (hunk 1 above)
Comment (theasianpianist) [reply tests/dualtor/test_normal_op.py:16]: Yes, I believe it was a typo introduced in a previous PR

**Pull request merged**